person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(name = 'unknown', age, parent_permission: true) # rubocop:todo Style/OptionalArguments
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :name, :id, :age
+
+  def is_of_age? # rubocop:todo Naming/PredicateName
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
* Kindly make this method a private one as stated in the [requirements ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Private%20method%20is_of_age%3F%20that%20returns%20true%20if%20%40age%20is%20greater%20or%20equal%20to%2018%20and%20false%20otherwise.)